pkg/render/crypto_utils.go

```diff
@@ -41,9 +41,9 @@ const (
 	blockTypeCert       = "CERTIFICATE"
 )
 
-// Creates a secret that will store the CA needed to generated certificates
+// VoltronTunnelSecret Creates a secret that will store the CA needed to generated certificates
 // for managed cluster registration
-func voltronTunnelSecret() *corev1.Secret {
+func VoltronTunnelSecret() *corev1.Secret {
```

Comment:
Since your sort of updating the usage of this could you change the name of this to be `GenerateVoltronTunnelSecret` or something like that so it's clear that a new certificate / key pair is being generated within this function.

Reply:
<none blocking>